### PR TITLE
bug/ads 649 refactor outline radius for button style radio group in safari

### PR DESCRIPTION
**Pull request by kaseyllabelle**

### CODE REVIEW

File: src/library/styles/components/_input-radio-button-group.scss [hunk 1]
```diff
@@ -1,39 +1,48 @@
...
-      @include button-default;
-      @include font-family-base-regular;
-      outline: space(xxs) solid $neutral-40;
+      display: inline-flex;
+      justify-content: center;
+      align-items: center;
+      min-width: space(8x);
       border-radius: space(3x);
-      border: $border-width-xs solid transparent;
+      border-width: space(xxs);
```
Comment (APJohns) [inline src/library/styles/components/_input-radio-button-group.scss:36]: We can use `$border-width-xxs` here

File: src/library/styles/components/_input-radio-button-group.scss [hunk 2]
```diff
@@ -1,39 +1,48 @@
...
+      border-color: inherit;
       padding: space(md) space(3x);
+      background: $neutral-100;
       color: $text-color-subtle;
-
-      &:hover,
-      &:active {
-        outline: space(xxs) solid $action-primary-20;
-      }
+      font-size: $ffs-body-assertive;
```
Comment (APJohns) [inline src/library/styles/components/_input-radio-button-group.scss:42]: Should this be `@include font-size($ffs-body-assertive)`?

File: src/library/styles/components/_input-radio-button-group.scss [hunk 3]
```diff
@@ -105,32 +117,29 @@
...
-            border-color: $action-primary-20;
-          }
-        }
-
         .bsds-input-radio {
-          &:checked {
+          &:not(:disabled) {
             + .bsds-input-radio-label {
-              border-color: $border-color-error;
+              border-color: $feedback-error-40;
```
Comment (APJohns) [inline src/library/styles/components/_input-radio-button-group.scss:123]: Thoughts on `$border-color-error` vs `$feedback-error-40`?

File: src/library/styles/components/_input-radio-button-group.scss (hunk 2 above)
Comment (kaseyllabelle) [reply src/library/styles/components/_input-radio-button-group.scss:42]: this one is the non-tokens library var for fixed font size

**Review by cblva — APPROVED** [review]

**Review by APJohns — APPROVED** [review]

**Pull request merged**